requirements.txt

```diff
@@ -1,13 +1,13 @@
 # Django, using PostgreSQL
-Django==1.9.6
+Django==1.9.10
```

Comment:
The pull request is updated. It looks like it is the right version. The [master](https://github.com/18F/ekip-api/blob/c1b20b40a3a10e9876e68be7537ed2709ff945b9/requirements.txt) also has 1.9.10.